index.js

```diff
@@ -1,10 +1,14 @@
 'use strict';
 
-module.exports = url => {
+module.exports = (url, options = {httpOnly: true}) => {
```

Comment:
This is not the correct way to do default values for options objects. If another option is added, it will silently not do the right thing as both options will be overwritten if one is set.

```suggestion
module.exports = (url, {httpOnly = true} = {}) => {
```